cmd/device-worker/main.go

```diff
@@ -162,17 +161,31 @@ func main() {
 		metricsStore,
 	)
 
+	dataDirPlaybook := path.Join(baseDataDir, "devicePlaybooks")
+	/* #nosec */
+	if err := os.MkdirAll(dataDirPlaybook, 0755); err != nil {
+		log.Fatal(fmt.Errorf("cannot create directory: %w", err))
+	}
+	ansibleManager, err := ansible.NewAnsibleManager(dispatcherClient, dataDirPlaybook)
+	if err != nil {
+		log.Fatalf("cannot start ansible manager, err: %v", err)
+	}
+	err = ansibleManager.ExecutePreviousPlaybooks()
+	if err != nil {
+		log.Fatalf("cannot run previous ansible playbooks, err: %v", err)
```

Comment:
If playbook failed here, should be stop the device? I'm not sure about TBH

Reply:
My idea here is: if one of the playbooks fails then the device is in an inconsistent state. That's why in my opinion the device should stop. WDYT?

Reply:
@jakub-dzon ^^^

Reply:
@masayag ATM is a single attempt. IMO, we can change this behavior once the playbook execution architecture will be defined, if necessary.

Reply:
`log.Fatalf` should be changed to `log.Errorf`. Fatalf will call `Exit(0)` and I don't think it is desired.

Reply:
@jakub-dzon yes, I forgot to change it! Thanks!